### PR TITLE
Rename LiteralInCondition cop to LiteralAsCondition

### PR DESCRIPTION
**Pull request by klssmith**
The `LiteralInCondition` cop has been renamed to `LiteralAsCondition` in
Rubocop version 0.51.0. Renaming this in configs/rubocop/other-lint.yml
prevents an unrecognized cop warning.

[Rubocop changelog](https://github.com/bbatsov/rubocop/blob/bed2a62e473fe5b6b26b3a572fb5432a7dc56b93/CHANGELOG.md#changes-1)

### CODE REVIEW

**Review by tijmenb — APPROVED** [review]

**Comment by klssmith** [comment]
@tijmenb Thanks for approving - I don't have write access to this repo, so I can't merge this or make Jenkins run against the PR.

**Comment by floehopper** [comment]
This fix was included in #89 which has been merged. Closing.

**Comment by tijmenb** [comment]
Sorry @klssmith! I missed this after the initial review - I'll give you & team access to this repo.

**Pull request closed without merging**
